src/kernels/MIOpenSubTensorOpWithScalarKernel.cl

```diff
@@ -61,6 +61,15 @@
 #endif
 #endif
 
+#if MIOPEN_USE_INT32 == 1
+#define _FLOAT unsigned
+#ifndef FLT_MAX
+#define MAX_VAL 4294967295 /* max value */
```

Comment:
max value should it be 2,147,483,647 for the int32 ?

Reply:
Oh i guess you are considering it unsigned.

Reply:
Well, in this specific case one 32 bit value represents four INT8's so max should be like `0x7f7f7f7f`. But it in unclear how can we meaningfully use it with 4xINT8 operands. And the MAX_VAL is not used at all in this OpWithScalar. Let me remove it.

BTW we use MAX_VAL in OpWithCastTensor (we saturate output to it in case of "overflow") and in some other primitives.

Reply:
Previous comment was wring; updated.